src/components/cards/featured-card.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { Thumbnail } from '../../interfaces/thumbnail';
+import { PillCategory } from '../../types/pill-category';
+import { TypographyScale } from '@mdb/flora';
```

Comment:
nitpick but can we have third party libraries like `@mdb/flora` at the top just like we do with React